translations/messages+intl-icu.de.yml

```diff
@@ -1828,6 +1828,7 @@ import.process.ready.for.review: "Bereit zur Überprüfung"
 import.process.generating.text: "In Verarbeitung (Text-Erkennung)"
 import.process.ready.for.approval: "Bereit für Bestätigung"
 import.process.finished: "Import abgeschlossen"
+import.statements: "Import (Stellungnahmen)"
```

Comment:
This is true for all projects but EWM. In EWM, segments can be imported as well.
I'd suggest to override the translation key in EWM to just say "Import" as it is now. To have a translation key that still makes sense in both situations, we could rename it to `import.menu` or something like that.

Reply:
Thanks, this makes sense to me and was done here https://github.com/demos-europe/demosplan-core/pull/2330/commits/ab92bb44a5c8f85f7894520e4d9b9a57334c9c7a 

and the ewm translation override PR can be found here: 
https://github.com/demos-europe/demosplan-project-ewm/pull/97